core/lib/protobuf_config/src/proto/config/prover.proto

```diff
@@ -14,8 +14,6 @@ message ProofCompressor {
   optional string universal_setup_path = 7; // required; fs path
   optional string universal_setup_download_url = 8; // required
   optional bool verify_wrapper_proof = 9; // required
-  optional string universal_fflonk_setup_path = 10; // required
```

Comment:
these should be added as reserved fields for the sake of backwards compatibility(you can look it up in other proto configs)